src/components/Article.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Article = ({ title, text, date }) => (
+  <article>
+    <h1>{title}</h1>
+    <span>
+      <i>{date}</i>
+    </span>
+    <p>{text}</p>
+  </article>
+);
+
+Article.propTypes = {
+  title: PropTypes.string.isRequired,
+  text: PropTypes.string.isRequired,
+  date: PropTypes.string.isRequired,
+};
```

Comment:
Let's make one of this props not required and use defaultProps for setting it's default value